content/en/blog/_posts/2024-10-01-migrate-cgroup-v2.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: blog
+title: 'The Shift to cgroup v2 in Kubernetes: What You Need to Know'
+date: 2024-10-01
+slug: kubernetes-cgroup-v2-shift
+author: >
+   Paco Xu (DaoCloud)
+---
+
+`cgroups` (control groups) are a Linux kernel feature used for managing system resources.
+Kubernetes uses cgroups to allocate resources like CPU and memory to containers,
+ensuring that applications run smoothly without interfering with each other.
+With the release of Kubernetes v1.31, cgroup v1 has been moved into maintenance mode.
```

Comment:
it's not very clear in the blog post what "maintenance mode" means.
maybe add a sentence or two after this one that explains it. in a new paragraph.

Reply:
The details are in another blog.
https://github.com/pacoxu/website/blob/b8bddc4aff4e9f67a35957dd07acce7b3d0ebdea/content/en/blog/_posts/2024-08-14-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md#L69-L76

Could we just add a link here?